verification/VerifyTasks.wdl

```diff
@@ -613,3 +613,53 @@ task CompareSnapTextFiles {
 }
 
 
+task CompareLibraryFiles {
+  input {
+    File test_text_file
+    File truth_text_file
+  }
+
+  command {
+    exit_code=0
+
+    a=~{test_text_file}
+    b=~{truth_text_file}
+
+    echo "Sorting files $a and $b"
+    sort "$a" > "a.sorted"
+    sort "$b" > "b.sorted"
+
+    echo "Calculating md5sums for $a and $b"
+    md5_a=$(md5sum "a.sorted" | cut -d ' ' -f1)
+    md5_b=$(md5sum "b.sorted" | cut -d ' ' -f1)
+
+    if [ $md5_a = $md5_b ]; then
+        echo "Files $a.sorted and $b.sorted have matching md5sums and are the same."
+      else
```

Comment:
```suggestion
    else
```

Reply:
Being super nit picky, but I think the indentation is a bit off (sometimes using 4 spaces and sometimes 2 spaces)